src/rtutils.c

```diff
@@ -604,6 +604,42 @@ JL_DLLEXPORT jl_value_t *jl_argument_datatype(jl_value_t *argt JL_PROPAGATES_ROO
     return (jl_value_t*)dt;
 }
 
+int is_globfunction(jl_value_t *v, jl_datatype_t *dv, jl_sym_t **globname_out) {
```

Comment:
Please add a newline before the `{` to match style. This could also be declared `static`.

Reply:
Oops! :) Thanks. done in latest push.